library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
-    * Temporal-slowing-elderly <nowiki> {suggestedTag=Brain-laterality, suggestedTag=Brain-region, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [Focal theta and/or delta activity over the temporal regions, especially the left, in persons over the age of 60. Amplitudes are low/similar to the background activity. Comment: focal temporal theta was found in 20 percent of people between the ages of 40-59 years, and 40 percent of people between 60 and 79 years. One third of people older than 60 years had focal temporal delta activity.]</nowiki>
-    * Breach-rhythm <nowiki> {suggestedTag=Brain-laterality, suggestedTag=Brain-region, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [Rhythmical activity recorded over cranial bone defects. Usually it is in the 6 to 11/sec range, does not respond to movements.]</nowiki>
-    * Other-uncertain-significant-pattern {requireChild}
+'''Uncertain-significant-pattern''' <nowiki>[EEG graphoelements or rhythms that resemble abnormal patterns but that are not necessarily associated with a pathology, and the physician does not consider them abnormal in the context of the scored recording (like normal variants and patterns). (Source: Beniczky ea 2013, Appendix S1; Beniczky ea 2017, Table 14.)]</nowiki>
+
+    * Sharp-transient-pattern <nowiki>{suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern}[Sharp transient. (Source: Beniczky ea 2017, Table 14.)]</nowiki>
+    * Wicket-spikes <nowiki>[Spike-like monophasic negative single waves or trains of waves occurring over the temporal regions during drowsiness that have an arcuate or mu-like appearance. These are mainly seen in older individuals and represent a benign variant that is of little clinical significance. (Source: Beniczky ea 2013, Appendix S6; Beniczky ea 2017, Table 14.)]</nowiki>
+    * Small-sharp-spikes <nowiki> {suggestedTag=Categorical-location-value, suggestedTag=Sensors, suggestedTag=Appearance-mode, suggestedTag=Discharge-pattern} [Benign epileptiform Transients of Sleep (BETS). Small sharp spikes (SSS) of very short duration and low amplitude, often followed by a small theta wave, occurring in the temporal regions during drowsiness and light sleep. They occur on one or both sides (often asynchronously). The main negative and positive components are of about equally spiky character. Rarely seen in children, they are seen most often in adults and the elderly. Two thirds of the patients have a history of epileptic seizures. (Source: Beniczky ea 2013, Appendix S6; Beniczky ea 2017, Table 14.)]</nowiki>
```

Comment:
In "Benign epileptiform Transients of Sleep (BETS)"
either epileptiform should start with a cap letter or Transients should be lower case.

Reply:
fixed